src/components/Dashboard/TodoList.vue

```diff
@@ -1726,7 +1727,7 @@ export default class TodoList extends Mixins(
 }
 
 .actions__more-actions {
-  padding: 0;
+  padding: 1;
```

Comment:
What units is this?

I _think_ that by default these are pixels, and if this is what you mean then please add the "px".

If, instead, you mean the equivalent of "pa-1", then note that the Vuetify spacing classes are 4px per unit.

If possible, please update this next time you're in here. Thanks.